.github/workflows/integration-linux.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         curl -fLO "https://github.com/bazelbuild/bazel/releases/download/${{ matrix.bazel_version }}/bazel-${{ matrix.bazel_version }}-installer-linux-x86_64.sh"
         chmod +x bazel-${{ matrix.bazel_version }}-installer-linux-x86_64.sh
         ./bazel-${{ matrix.bazel_version }}-installer-linux-x86_64.sh --user
-        echo ::add-path::${HOME}/bin
```

Comment:
why isnt `${HOME}` not set correctly?

Reply:
I'm not sure why I thought this was the issue. I checked and `${HOME}` resolves to `/home/runner` which is what I wanted. I thought that `bazel` wasn't resolving to my installed binary because this exact same version succeeded on `macOS`. Later I ran it against a range of `bazel` versions from `2.x` to `3.4.1` but all of them failed the `skaffold diagnose` test for `bazel`